test/client_scan_test.py

```diff
@@ -172,16 +172,6 @@ def test_metadata_search(self):
         result = list(api.search_by_metadata('hash.sha256:275a021bbfb6489e54d471899f7db9d1663fc695ec2fe2a2c4538aabf651fd0f'))
         assert result[0].sha256 == '275a021bbfb6489e54d471899f7db9d1663fc695ec2fe2a2c4538aabf651fd0f'
 
-    @responses.activate
```

Comment:
Please provide a valid json that can be parsed with the changes you made. Removing test in client-facing packages should be avoided.